spec/services/builders/distribution/pull_requests/time_to_second_review_repository_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.describe Builders::Distribution::PullRequests::TimeToSecondReviewRepository do
+  describe '.call' do
+    let(:user1) { create(:user, login: 'user1') }
+    let(:user2) { create(:user, login: 'user2') }
+    let(:user3) { create(:user, login: 'user3') }
+
+    let(:repository_ok) { create(:repository) }
```

Comment:
what does "ok" mean here? can we think of another naming for this to make it more clear? 🤔